keadm/app/cmd/cloud/init.go

```diff
@@ -62,13 +61,13 @@ func NewCloudInit(out io.Writer, init *types.InitOptions) *cobra.Command {
 		Short:   "Bootstraps cloud component. Checks and install (if required) the pre-requisites.",
 		Long:    cloudInitLongDescription,
 		Example: cloudInitExample,
-		Run: func(cmd *cobra.Command, args []string) {
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
why is this renamed as RunE ?

Reply:
Run and RumE are separate fields of the struct. See the Cobra documentation.
I guess E stands for error.